Renewal menu structures

caramis opened the pull request.

jhlee-young commented:
`develop/guides/start-the-light-client-daemon.md:37`
`full-node/run-a-full-xpla-node/configure-general-settings.md:58`

caramis commented:
@jhlee-young fixed !

jhlee-young reviewed (approved):
LGTM

caramis committed:
feat: separate "Learn"

caramis committed:
feat: seperate "Develop"

caramis committed:
feat: seperate "Full Node"

caramis committed:
feat: seperate "Guide"

caramis committed:
feat: X instead of Twitter

caramis committed:
feat: introduce "Ecosystem"

caramis committed:
fix: content of "confirm your validator is active"

caramis committed:
fix: miscellaneous

caramis committed:
feat: adjust table background color

The variable is not actually applied, but the background color is
initialized, resulting in the same effect.

caramis committed:
feat: XPLA iamges
